Style.css

```diff
@@ -76,3 +76,139 @@ li {
 .union {
   color: #6070ff;
 }
+
+.container {
+  display: grid;
+  height: 100vh;
+  grid-template-columns: 1fr 3fr;
+  place-items: center;
+}
+
+#portfolio {
+  display: grid;
+  gap: 88px;
+  padding: 24px;
+  background: #e5e5e5;
+}
+
+.portfolio-work {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  box-sizing: border-box;
+  padding: 16px;
+  gap: 12px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.work-title {
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.job-description {
+  display: flex;
+  gap: 24px;
+}
+
+.work-technologie {
+  display: flex;
+  gap: 24px;
+}
+
+.button-btn {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  border-radius: 3px;
+}
+
+.header-img {
+  height: 700px;
+  display: flex;
+  flex-direction: column;
+  padding-left: 5%;
+  background-image: url("./assets/Background.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  padding-top: 20%;
+  background-color: #7f8cff;
+  border-radius: 70px 0% 0% 0%;
+}
+
+.index {
+  margin: 2px;
+  padding: 24px;
+}
+
+.button {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+  display: flex;
+  border-radius: 10px;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  margin: 0;
+  margin-right: 24px;
+  padding: 10px 20px;
+  background-color: #fff;
+  width: fit-content;
+  border: solid 2px #6070ff;
+}
+
+.form-group {
+  display: grid;
+  padding: 15px;
+  padding-bottom: 0;
+  border-radius: 10px;
+}
+
+.textarea-work {
+  display: grid;
+  padding: 15px;
+  text-align: center;
+}
+
+.pl-0 {
+  padding-left: 0;
+}
+
+.index-h1 {
+  text-align: center;
+}
+
+.field-spacing {
+  padding: 10px 20px;
+  border-radius: 10px;
+}
+
+.language-layout {
+  display: flex;
+  flex-direction: column;
+}
+
+.language-content {
+  display: flex;
+  flex-direction: row;
+  background-color: #808080;
+  margin-bottom: 10px;
+  padding: 10px;
+}
+
+.bottom-line {
+  border-bottom: solid 2px #000;
+}
```

Comment:
- Kindly, stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=C0vU075hnDfZLPwM-0) the color, border and gaps 👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/206512754-da2c5348-5ffc-4652-9399-c6bebf7a1dd9.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/206512767-38033bee-a1c6-49c1-ad6b-e89578828d4d.png" width="200">|


😎️